osquery/tables/system/windows/registry.cpp

```diff
@@ -29,6 +31,7 @@
 #include "osquery/core/conversions.h"
 #include "osquery/filesystem/fileops.h"
 #include "osquery/tables/system/windows/registry.h"
+#include <osquery/filesystem.h>
```

Comment:
Re-order this into the block above, which also share the `<` `>` search preference.